src/assets/styles/colors.scss

```diff
@@ -1 +1,6 @@
-// define app colors
+$primary: #3F51B5;
+$white: #ffffff;
+$buttonConfirm: #82B1FF;
+$button: #3F51B5;
+$alertError: #FF5252;
```

Comment:
There is `alert` or `error`, naming something `alertError` does not make much sense. Same for `successError` -> something can't be both a success and an error.